packages/core/src/helpers.ts

```diff
@@ -212,11 +218,18 @@ export const truncateString = (
 // Is the file coming from the injection plugin?
 export const isInjectionFile = (filename: string) => filename.includes(INJECTED_FILE);
 
+// From a bundler's name, is it part of the "xpack" family?
+export const isXpack = (bundlerName: BundlerFullName) =>
+    ['rspack', 'webpack4', 'webpack5', 'webpack'].includes(bundlerName);
+
 // Replacing fs-extra with local helpers.
 // Delete folders recursively.
 export const rm = async (dir: string) => {
     return fsp.rm(dir, { force: true, maxRetries: 3, recursive: true });
 };
+export const rmSync = async (dir: string) => {
```

Comment:
```suggestion
export const rmSync = (dir: string) => {
```

Reply:
(maybe some usages to update with `await` to be removed? though unlikely)

Reply:
Oh, nice catch !